libV2/utils/index.js

```diff
@@ -1,9 +1,11 @@
 const log = require('./log');
 const shard = require('./shard');
 const timestamp = require('./timestamp');
+const func = require('./func');
 
 module.exports = {
     ...log,
     ...shard,
     ...timestamp,
+    ...func,
```

Comment:
Should these be destructured here?

Reply:
I think it would be easier to read if it is `util.func.asyncOrCallback` than `util.asyncOrCallback` no?

Reply:
`utils` is normally used like `const { LoggerContext, asyncOrCallback } = require('./utils');` so I think it make sense to export them under a single namespace. Additionally it would make `func` different than the other submodules and I like to keep things consistent.